mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/9 - object selection

Closes #9 

To test:
- Play introductory scenario
- Click on user and see user name displayed on right side of screen (further properties will be in a separate case)
- Buy a computer for Curly and click on it to see computer name displayed on right side of screen (see above paranthetical)
- Click somewhere else to deselect object

Notes:
- Each object types' property panel will get its own case to implement them separately
- OnGUI ContextMenu appears when you select an object and goes away when deselecting it, which will eventually be replaced completely with other UI functionality
